File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/TiuNotePreview.java
package gov.va.med.mhv.sm.model;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class TiuNotePreview {
	
	private boolean isLocked;
	private Patient patient;
	private Date createdDate;
	private String facilityName;
	private boolean isAddendum;
	private String existingNote;
	private String proposedNote;
	private List<Message> existingMessages;
	private List<Message> proposedMessages;
	
	public TiuNotePreview(){
		existingMessages = new ArrayList<Message>();
		proposedMessages = new ArrayList<Message>();
	}
	
	public Patient getPatient() {
		return patient;
	}
	public void setPatient(Patient patient) {
		this.patient = patient;
	}
	public Date getCreatedDate() {
		return createdDate;
	}
	public void setCreatedDate(Date createdDate) {
		this.createdDate = createdDate;
	}
	public String getFacilityName() {
		return facilityName;
	}
	public void setFacilityName(String facilityName) {
		this.facilityName = facilityName;
	}
	public boolean isAddendum() {
		return isAddendum;
	}
	public void setAddendum(boolean isAddendum) {
		this.isAddendum = isAddendum;
	}
	public String getExistingNote() {
		return existingNote;
	}
	public void setExistingNote(String existingNote) {
		this.existingNote = existingNote;
	}
	public String getProposedNote() {
		return proposedNote;
	}
	public void setProposedNote(String proposedNote) {
		this.proposedNote = proposedNote;
	}
	public List<Message> getExistingMessages() {
		return existingMessages;
	}
	public void setExistingMessages(List<Message> existingMessages) {
		this.existingMessages = existingMessages;
	}
	public List<Message> getProposedMessages() {
		return proposedMessages;
	}
	public void setProposedMessages(List<Message> proposedMessages) {
		this.proposedMessages = proposedMessages;
	}
	public boolean isLocked() {
		return isLocked;
	}
	public void setLocked(boolean isLocked) {
		this.isLocked = isLocked;
	}
	
	
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/TiuNotePreview.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    public class TiuNotePreview
    {
        private bool _isLocked;

        public bool IsLocked
        {
            get { return _isLocked; }
            set { _isLocked = value; }
        }
        private gov.va.medora.mdo.domain.sm.Patient _patient;

        public gov.va.medora.mdo.domain.sm.Patient Patient
        {
            get { return _patient; }
            set { _patient = value; }
        }
        private DateTime _createdDate;

        public DateTime CreatedDate
        {
            get { return _createdDate; }
            set { _createdDate = value; }
        }
        private string _facilityName;

        public string FacilityName
        {
            get { return _facilityName; }
            set { _facilityName = value; }
        }
        private bool _isAddendum;

        public bool IsAddendum
        {
            get { return _isAddendum; }
            set { _isAddendum = value; }
        }
        private string _existingNote;

        public string ExistingNote
        {
            get { return _existingNote; }
            set { _existingNote = value; }
        }
        private string _proposedNote;

        public string ProposedNote
        {
            get { return _proposedNote; }
            set { _proposedNote = value; }
        }
        private List<Message> _existingMessages = new List<Message>();

        public List<Message> ExistingMessages
        {
            get { return _existingMessages; }
            set { _existingMessages = value; }
        }
        private List<Message> _proposedMessages = new List<Message>();

        public List<Message> ProposedMessages
        {
            get { return _proposedMessages; }
            set { _proposedMessages = value; }
        }
    }
}
